sdk/include/ndk/umfuncs.h

```diff
@@ -178,7 +178,7 @@ LdrQueryImageFileExecutionOptions(
     _In_ PUNICODE_STRING SubKey,
     _In_ PCWSTR ValueName,
     _In_ ULONG ValueSize,
-    _Out_ PVOID Buffer,
+    _Out_opt_ PVOID Buffer,
     _In_ ULONG BufferSize,
```

Comment:
```suggestion
    _In_opt_ ULONG BufferSize,
```

Reply:
`_Opt_` is useless for non-pointer params